docs/how-to/spack-intro.md

```diff
@@ -57,252 +64,297 @@ Reference: <https://spack-tutorial.readthedocs.io/en/latest/>
 | **hipsolver**             | hipsolver              |
 | **mlirmiopen**            | mlirmiopen             |
 
-One need to install all the pre-requisites before performing the SPACK installation.
+You must install all prerequisites before installing SPACK.

```

Comment:
Should we highlight this as a note or 'important,' as it is a critical step? 

@LisaDelaney

Reply:
@Rmalavally I'm on the fence as to whether we really need that statement at all...by definition, prerequisites come first.

Reply:
Installation steps do require frequent reminders, given the complexity. I am okay with keeping it as a note (a gentle reminder).